resources/common/plugins/macro/batch-edit-oto-parameter/batch-edit-oto-parameter.js

```diff
@@ -5,7 +5,7 @@ let keepDistance = params["keepDistance"]
 let nameTexts = [
     ["offset", ["Offset", "左边界", "左ブランク"]],
     ["fixed", ["Fixed", "固定", "固定範囲"]],
-    ["overlap", ["Overlap", "重叠"]],
+    ["overlap", ["Overlap", "重叠", "オーバーラップ"]],
```

Comment:
Thank you for the fix. Could you also bump up the version of this plugin? Otherwise it's not updated